src/libs/actions/Report.js

```diff
@@ -875,6 +884,21 @@ function handleReportChanged(report) {
     // Store the max sequence number for each report
     reportMaxSequenceNumbers[report.reportID] = report.maxSequenceNumber;
 
+    // Store the max sequence number for each report excluding loading actions
+    Onyx.connect({
```

Comment:
This is basically a memory leak as it's performed inside the callback of `handleReportChanged()`. We run this anytime a report changes. So we will add tons and tons of subscriptions here over the lifetime of a session.

Reply:
We may be able to get rid of the whole thing as per your previous comment. If we don't, we can try and figure out a better place to put it :)